docs/swagger.yaml

```
@@ -1,6 +1,29 @@
+definitions:
+  model.Tmodel:
+    type: object
 info:
   contact: {}
 paths:
+  /test:
+    post:
+      consumes:
+      - application/json
```

YAMLlint (1.35.1)

[error] wrong indentation: expected 8 but found 6

(indentation)